travis/install_pg

```diff
@@ -4,13 +4,38 @@
 
 set -eux
 
-# always install postgresql-common
-packages="postgresql-common libedit-dev libpam0g-dev libselinux1-dev"
+# If USE_CUSTOM_PG_BUILD variable is not set, we will assume custom build is
+# not required. If custom build is not required, we will install PostgreSQL
+# via apt, otherwise we will compile from the source.
+if [ "${USE_CUSTOM_PG_BUILD-0}" = "0" ]; then
```

Comment:
Rather than adding conditionals and indenting everything, could you just add a new script (maybe `install_custom_pg`) and make each script exit early based on `USE_CUSTOM_PG_BUILD`'s value?